collect/src/marinade_service.rs

```diff
@@ -31,6 +31,28 @@ pub fn get_marinade_stakes(
     )?)
 }
 
+pub fn get_institutional_stakes(
+    rpc_client: &RpcClient,
+    epoch: Epoch,
+    stake_history: &StakeHistory,
+) -> anyhow::Result<HashMap<String, u64>> {
+    let mut institutional_authority = "STNi1NHDUi6Hvibvonawgze8fM83PFLeJhuGMEXyGps".try_into()?;
+
+    let institutional_stakes = get_stakes_groupped_by_validator(
+        rpc_client,
+        &institutional_authority,
+        None,
+        epoch,
+        stake_history,
+    )?;
+
+    assert!(
+        !institutional_stakes.is_empty(),
+        "Failed to fetch institutional stake data"
```

Comment:
Are we sure this is correct to expect there will be always some institutional stake?